README.md

```diff
@@ -343,7 +358,10 @@ docker-compose ps
 
 ### Purpose & Usage
 
-- The DOCKER_HOST_IP environment variable is used to communicate with the bootstrap server that the instance of Kafka is running on.
+- The `DOCKER_HOST_IP` environment variable is used to communicate with the bootstrap server that the instance of Kafka is running on.
+- The `MAVEN_GITHUB_TOKEN_NAME` environment variable is the name of the generated token from xxx used for pulling the jpo-ode java image.

```

Comment:
It might be good to specify these new environment variables as specifically required to run the jpo-geojsonconverter in Docker. If someone wanted to run an already built image in K8s, for example, they wouldn't need to set these - right?

Reply:
Yeah, good point. I'll adjust the documentation accordingly.

Reply:
addressed

Reply:
Nice, I like the clarity.